pyomo/contrib/mindtpy/tests/test_mindtpy_feas_pump.py

```diff
@@ -23,7 +23,7 @@
 from pyomo.contrib.mindtpy.tests.feasibility_pump1 import Feasibility_Pump1
 from pyomo.contrib.mindtpy.tests.feasibility_pump2 import Feasibility_Pump2
 
-required_solvers = ('ipopt', 'cplex')
+required_solvers = ('ipopt', 'glpk', 'cplex')
 if all(SolverFactory(s).available() for s in required_solvers):
     subsolvers_available = True
```

Comment:
This will skip some of the tests of CPLEX is not available although mots of them run with GLPK.